### PR TITLE
Avoid saving charts to filesystem

### PR DESCRIPTION
**Pull request by valfirst**

### CODE REVIEW

**Review by ikalinin1 — APPROVED** [review]

**Comment by svasenkov** [comment]
Why not? I think, to save chart localy is not bad idea. Maybe add it as optional parameter?

**Comment by valfirst** [comment]
> Why not? I think, to save chart localy is not bad idea. Maybe add it as optional parameter?

potential issues:
- permissions management (java-process may be run under user who don't have permissions to write to filesystem or to the folder where the process is running from)
- performance: IO operations loading data from FS and flushing data to FS (this point could be considered as insignificant for this utility, but if there is a chance to avoid peformance drawbacks, why not to do it)
- differences in FS management for different OS and environments (e.g., containerised infrastructure) - however that's not _yet_ an issue for the library
- non-transparent logic: it's hard to follow the library logic when file is created in one part of the project and then referenced by its name in another part - however it's a project specific issue and it could be resolved in another way
